tutorials/matching_fof.ipynb

```diff
@@ -366,7 +357,7 @@
    "source": [
     "# load redshift and shear parameters from the extragalactic catalog, only for galaxise that are already in `matched_gals`\n",
     "extragalactic_data = extragalactic_cat.get_quantities(\n",
-    "    ['galaxy_id', 'mag_i', 'ellipticity_true', 'shear_1', 'shear_2'],\n",
+    "    ['galaxy_id', 'mag_i_lsst', 'ellipticity_true', 'shear_1', 'shear_2'],\n",
     "    filters=[(lambda x: np.in1d(x, matched_gals['object_id'].values, True), 'galaxy_id')]\n",
```

Comment:
Most of the changes seem simple, but I suppose one check would be that this redefinition of mag_i_lsst is backwards propogated?